src/vstest.console/Internal/ConsoleLogger.cs

```diff
@@ -371,6 +379,39 @@ private static void DisplayFullInformation(TestResult result)
             }
         }
 
+        /// <summary>
+        /// Returns the parent Execution id of given test result. 
+        /// </summary>
+        /// <param name="testResult"></param>
+        /// <returns></returns>
+        private Guid GetParentExecutionId(TestResult testResult)
+        {
+            var parentExecutionIdProperty = testResult.Properties.FirstOrDefault(property =>
+                property.Id.Equals(ParentExecutionIdPropertyIdentifier));
+            return parentExecutionIdProperty == null
```

Comment:
GetPropertyValue itself should take care of this.

Reply:
`GetPropertyValue` will throw if the property is null. This line is imho needed.